pyphare/pyphare/simulator/monitoring.py

```
@@ -13,6 +13,29 @@
 logger = getLogger(__name__)
 
 
+@dataclass
```

Ruff (0.15.15)

[error] Undefined name `dataclass`

(F821)